lecture2gether-vue/src/views/Room.vue

```diff
@@ -69,14 +73,7 @@ export default class L2gPlayerView extends Vue {
         width: 100%!important;
     }
     .l2g-player {
-        margin: auto;
-        //hacky way to force centered video.
-        //this size has to be the same as the video
-        //width in src/components/Player.vue
+        width: 100vw;
         max-width: 750px;
```

Comment:
How about 1000px? I think that looks better (but it depends on the screen size, I guess)

Reply:
px are always device-pixel so they should be resolution independent. regardless, we should test it

Reply:
We discussed this, it will possibly be changed in a future pull request.